src/cz/crcs/ectester/reader/test/CardTestVectorSuite.java

```diff
@@ -52,6 +52,9 @@ protected void runTests() throws Exception {
          */
         Map<String, EC_KAResult> results = EC_Store.getInstance().getObjects(EC_KAResult.class, "test");
         for (EC_KAResult result : results.values()) {
+            if("DH_PLAIN".equals(result.getKA())) {
```

Comment:
This is now unnecessary I think. What will happen with the cards is that the applet will try to allocate an `ALG_ECDH_PLAIN` key agreement, which will fail if it is not supported or succeed if it is. Both behaviors are fine and in case it is supported we want to test it as well, so please drop this check.

Reply:
I wasn't sure if this behavior was correct or whether the tests for plain ECDH should be implemented in some other way, so I left it in there just in case. This should now be resolved.